src/file_analysis/file_analysis.bif

```diff
@@ -41,7 +41,7 @@ function Files::__set_reassembly_buffer%(file_id: string, max: count%): bool
 ## :zeek:see:`Files::enable_analyzer`.
 function Files::__enable_analyzer%(tag: Files::Tag%) : bool
 	%{
-	auto c = zeek::file_mgr->Lookup(tag->AsEnumVal());
+	auto c = zeek::file_mgr->Lookup(tag->AsEnumVal(), false);
```

Comment:
Similar argument as for above, I believe there should be a way to transparently toggle all mapped analyzers.

Reply:
Same answer here too.